test/TestProjects/MgmtSafeFlatten/Generated/Models/TypeFour.cs

```diff
@@ -22,7 +22,7 @@ internal TypeFour()
         /// <summary> The id of layer one. </summary>
         public string LayerOneUniqueId
```

Comment:
This is non-nullable because we don't expose nullable reference types in Mgmt plane APIs?